ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -1,18 +1,20 @@
 # coding=utf-8
 from __future__ import absolute_import
+
 import re
 import string
+from six.moves import range
 
+from language_utilities.constant import (ENGLISH_LANG, INDIC_LANGUAGES_SET, EUROPEAN_LANGUAGES_SET)
```

Comment:
Our adding of more previous messages will only avoid false positives. To enable detection we will have to update the logic in `detect_entity` and whatever it calls from that point onwards